### PR TITLE
updated para 1

### PR DESCRIPTION
**Pull request by harisree06**

### CODE REVIEW

File: sampl.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Fighting evil by ______, winning love by ______
-Never running from a ______, she is the one named ______.
+Fighting evil by moonlight, winning love by daylight
```
Comment (harisree06) [inline sampl.txt:1]: Good

**Comment by goldbergyossi** [comment]
amazing

File: sampl.txt (hunk 1 above)
Comment (Ari-Adaleg) [reply sampl.txt:1]: **WHY!!!!**

File: sampl.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+Never running from a friend, she is the one named Sailor moon.
```
Comment (TSAGI) [inline sampl.txt:2]: capitalize Moon

**Comment by Ari-Adaleg** [comment]
*Don't know why I'm commenting...*

**Pull request merged**